<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voiceUserWidget</class>
 <widget class="QWidget" name="voiceUserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>44</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>44</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#voiceUserWidget { color: rgb(255,255,255); background-color: rgba(34,34,34,191); }
QHBoxLayout#horizontalLayout { background-color: rgba(34,34,34,191); }
QCheckBox#muteCheckBox {color: rgb(255,255,255);}
QCheckBox#muteCheckBox {background-color: rgba(34,34,34,191);}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="avatarNameLabel">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Avatar Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="muteCheckBox">
       <property name="text">
        <string>Mute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
